### ISSUE TITLE
H3lix uicache failed

### ISSUE THREAD

**Issue by TheRasPiGuy** [pixelomer/AltDeploy#30]
Can't install H3lix because uicache failed, looked it up says you need to sign without the "get-task-allow" entitlement

**Comment by pixelomer** [comment]
It's a DRM, I didn't add a patcher yet

**Comment by stidch** [comment]
I face the same issue here.

**Comment by jonathanpow** [comment]
Yes, I'm having exactly the same issue. Loaded onto my iPad via AltDeploy using a virtual machine. unc0ver loads perfectly via the same system and method on my iPhone.

**Comment by burgermoritz** [comment]
I have the same problem with my iPad (4th Generation), when trying to jailbreak. Would love to be able to jailbreak.

**Comment by mstevo58** [comment]
any update on this issue... is there a fix?

**Comment by CaseyCZ** [comment]
I have the same problem.any update? Thanks

**Comment by mstevo58** [comment]
Doing some reading over the last few days..casyCZ, I thinking is it iOS-signer and Xcode with a patch I believe, trying to find tutorial.

**Comment by tenwx** [comment]
Found a way to patch H3lix RC6. https://gist.github.com/jakeajames/b44d8db345769a7149e97f5e155b3d46

**Comment by burgermoritz** [comment]
Thank you very very much

**Comment by mstevo58** [comment]
Fantastic...thanks heaps, up and running...

**Comment by jonathan3195** [comment]
> Found a way to patch H3lix RC6. https://gist.github.com/jakeajames/b44d8db345769a7149e97f5e155b3d46

Any chance you can help me how to do it

**Comment by jonathan3195** [comment]
> Fantastic...thanks heaps, up and running...

How

**Comment by CaseyCZ** [comment]
> > Fantastic...thanks heaps, up and running...
> 
> How

Download the patch.sh
Extract it.
Copy it the patch.sh to the desktop
Copy the iPA to the desktop
Open a terminal and run cd ~/Desktop
Run chmod +x patch.sh
Run ./patch.sh h3lix-RC6.ipa h3lix-RC6-patch.ipa
Follow this to sign it
Jailbreak.

**Comment by jonathan3195** [comment]
H I thanks! Someone on twitter already had the patch ipa so they sent me it
:)

**Comment by MiladShojaei** [comment]
link for patch version :
https://workupload.com/file/etd9TYxN

**Comment by rtcatc** [comment]
> 修补程序版本的链接：[https](https://workupload.com/file/etd9TYxN) :
> [//workupload.com/file/etd9TYxN](https://workupload.com/file/etd9TYxN)

thanks!

**Comment by ghost** [comment]
> link for patch version :
> https://workupload.com/file/etd9TYxN

Bless you, good sir for getting this uploaded.
Shame on tihmstar for his DRM thing in h3lix RC6 or below (not including RC1, however)
